site/installing-weave.md

```diff
@@ -32,6 +32,8 @@ With Weave Net downloaded onto your VMs or hosts, you are ready to launch a Weav
 
 CoreOS users see [here](/guides/networking-docker-containers-with-weave-on-coreos/) for an example of installing Weave using cloud-config.
 
+Amazon EC2 users may want to use [AWS-VPC mode](/site/using-weave/awsvpc.md)
```

Comment:
I am concerned that EC2 users will jump there as a matter of course. The AWS-VPC mode is a fair bit harder to set up than sleeve/fastdp, so, IMO we should steer users towards the latter first and only recommend AWS-VPC for users who reach performance limits.

Also, this is missing a full stop.